simpleviewinc/keg-hub
# Remove stylesParser from ReTheme

**Ticket**: [ZEN-446](https://jira.simpleviewtools.com/browse/ZEN-446)

## Context

* Removed the styleParser from retheme

## Goal

* Removed the styleParser from retheme, no longer needed

## Updates

* `repos/re-theme/configs/rollup.config.js`
  * Removed the styles parser export
* `repos/re-theme/src/styleParser/*`
  * Remove all styleParser related files

## Testing

* Run this docker package url =>`keg evf pack run docker.pkg.github.com/simpleviewinc/keg-packages/tap:zee-446-react-strap-modal`

* Navigate to this url in the browser => http://evf-zee-446-react-strap-modal.local.kegdev.xyz/
  > The tap is using this updated version of re-theme, with the styles parser removed

* You may see some errors when the docker package is starting.
  * They look like this => http://snpy.in/6roPOD
  * These errors are not related and can be ignored
  * I'm testing updating to Node 14, and there a number of changes that cause these error to show up
  * They do not affect the actual application running

* **Ensure**
  * no errors are thrown within the app
  * the app run as expected
* **IMPORTANT** - The modal styles are expected to be incorrect
  * This is due to the updates to the tap's modal using `boot-strap`
  * There is a seperate ticket to fix these issues
